src/test/kotlin/de/bund/digitalservice/useid/statics/GlobalStaticErrorIntegrationTest.kt

```diff
@@ -0,0 +1,91 @@
+package de.bund.digitalservice.useid.statics
```

Comment:
Feels like there should also be a test for 500 error, don't you think?

Reply:
I tried to throw error but for any request with html accept header, Spring already prevents to throw the error we defined in the controller and returns 406 instead